src/main/java/su/terrafirmagreg/core/mixins/common/gtceu/GTMultiMachinesMixin.java

```diff
@@ -0,0 +1,49 @@
+package su.terrafirmagreg.core.mixins.common.gtceu;
+
+import com.gregtechceu.gtceu.GTCEu;
+import com.gregtechceu.gtceu.api.data.RotationState;
+import com.gregtechceu.gtceu.api.machine.MultiblockMachineDefinition;
+import com.gregtechceu.gtceu.api.pattern.FactoryBlockPattern;
+import com.gregtechceu.gtceu.common.data.machines.GTMultiMachines;
+import com.gregtechceu.gtceu.common.machine.multiblock.primitive.CharcoalPileIgniterMachine;
+import net.minecraft.world.level.block.Blocks;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import su.terrafirmagreg.core.common.data.TFGTags;
+
+import static com.gregtechceu.gtceu.api.pattern.Predicates.*;
+import static com.gregtechceu.gtceu.common.data.GTBlocks.BRONZE_HULL;
+import static com.gregtechceu.gtceu.common.data.GTRecipeTypes.DUMMY_RECIPES;
+import static com.gregtechceu.gtceu.common.registry.GTRegistration.REGISTRATE;
+
+@Mixin(value = GTMultiMachines.class, remap = false)
+public abstract class GTMultiMachinesMixin {
+
+    /**
+     * Исправляет странные требования для мультиблока (земля -> земля тфк, дуб -> дерево тфк)
+     * */
+    @Shadow
+    public static final MultiblockMachineDefinition CHARCOAL_PILE_IGNITER = REGISTRATE
```

Comment:
where did this come from? I think this might be a default gtceu thing to get lots of charcoal for its own primitive blast furnace, whereas we have tfc's system for charcoal instead